src/components/structures/ContextualMenu.js

```diff
@@ -215,16 +216,22 @@ export default class ContextualMenu extends React.Component {
             menuStyle["paddingRight"] = props.menuPaddingRight;
         }
 
+        const wrapperStyle = {};
+        if (!isNaN(Number(props.zIndex))) {
```

Comment:
Since you are checking for numbers in `propTypes`, wouldn't `if (props.zIndex)` be enough?

Reply:
This is kinda for the extremely rare case of when someone uses zero, and is blatant copy/paste from above.

Reply:
Hmm, so it guards against `0`? Or do you mean some kind of string value...?

Reply:
`if (props.zIndex)` will fail if zero.

Reply:
Oh, haha, fair enough. 😅 Seems fine, plus matches the surrounding code.